app/common/src/utilities/data/__tests__/iterator.test.ts

```diff
@@ -1,5 +1,17 @@
 import { expect, test } from 'vitest'
-import * as iter from '../iter'
+
+import {
+  chain,
+  count as countFn,
+  empty,
+  filter,
+  filterDefined,
+  last as lastFn,
+  range,
+  tryGetSoleValue,
+  zip,
+  zipLongest,
+} from '../iter'
```

Comment:
Why this change? We usually use the `iter` module by qualified import, because it defines a lot of names that are likely to collide with locals.

Reply:
oh... yeah, my bad. i guess ideally we could record exceptions (?) like these somewhere? just so there's somewhere to refer to when doing refactors